ros_2_cal:
      neck_pitch : 1
      neck_yaw: 1
      l_shoulder_pitch: 1
      l_elbow_yaw: 1
      l_shoulder_roll: 1
      l_elbow_pitch: 1
      l_wrist_yaw: 1
      r_shoulder_pitch: 1
      r_elbow_yaw: 1
      r_shoulder_roll: 1
      r_elbow_pitch: 1
      r_wrist_yaw: 1
      waist_pitch: 1
      waist_yaw: 1
      l_hip_pitch: 1
      l_hip_yaw: 1
      l_hip_roll: 1
      l_knee_pitch: 1
      l_ankle_pitch: 1
      l_ankle_roll: 1
      r_hip_pitch: 1
      r_hip_yaw: 1
      r_hip_roll: 1
      r_knee_pitch: 1
      r_ankle_pitch: 1
      r_ankle_roll: 1

robot_2_cal:
      neck_pitch : 1
      neck_yaw: 1
      l_shoulder_pitch: -1
      l_elbow_yaw: -1
      l_shoulder_roll: 1
      l_elbow_pitch: -1
      l_wrist_yaw: -1
      r_shoulder_pitch: 1
      r_elbow_yaw: -1
      r_shoulder_roll: 1
      r_elbow_pitch: 1
      r_wrist_yaw: -1
      waist_pitch: -1
      waist_yaw: 1
      l_hip_pitch: 1
      l_hip_yaw: 1
      l_hip_roll: 1
      l_knee_pitch: -1
      l_ankle_pitch: -1
      l_ankle_roll: -1
      r_hip_pitch: -1
      r_hip_yaw: 1
      r_hip_roll: 1
      r_knee_pitch: 1
      r_ankle_pitch: 1
      r_ankle_roll: -1

